mmv1/products/vertexai/api.yaml

```diff
@@ -277,6 +277,59 @@ objects:
                   Configuration of the snapshot analysis based monitoring pipeline running interval. The value is rolled up to full day.
 
                   A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
+              - !ruby/object:Api::Type::Integer
+                name: 'monitoringIntervalDays'
+                description: |
+                  Configuration of the snapshot analysis based monitoring pipeline running interval. The value indicates number of days. The default value is 1.
+                  If both FeaturestoreMonitoringConfig.SnapshotAnalysis.monitoring_interval_days and [FeaturestoreMonitoringConfig.SnapshotAnalysis.monitoring_interval][] are set when creating/updating EntityTypes/Features, FeaturestoreMonitoringConfig.SnapshotAnalysis.monitoring_interval_days will be used.
+                default_value: 1
+              - !ruby/object:Api::Type::Integer
+                name: 'stalenessDays'
+                description: |
+                  Customized export features time window for snapshot analysis. Unit is one day. The default value is 21 days. Minimum value is 1 day. Maximum value is 4000 days.
+                default_value: 21
+          - !ruby/object:Api::Type::NestedObject
+            name: 'importFeaturesAnalysis'
+            description: |
+              The config for ImportFeatures Analysis Based Feature Monitoring.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: state
+                description: |
+                  Whether to enable / disable / inherite default hebavior for import features analysis. The value must be one of the values below:
+                  * STATE_UNSPECIFIED: Should not be used.
+                  * DEFAULT: The default behavior of whether to enable the monitoring. EntityType-level config: disabled.
+                  * ENABLED: Explicitly enables import features analysis. EntityType-level config: by default enables import features analysis for all Features under it.
+                  * DISABLED: Explicitly disables import features analysis. EntityType-level config: by default disables import features analysis for all Features under it.
+              - !ruby/object:Api::Type::String
+                name: 'anomalyDetectionBaseline'
+                description: |
+                  Defines the baseline to do anomaly detection for feature values imported by each [entityTypes.importFeatureValues][] operation. The value must be one of the values below:
+                  * BASELINE_UNSPECIFIED: Should not be used.
```

Comment:
Should we maybe not list this if it shouldn't be used?

Reply:
The same here. I'll remove it. Thanks!